example-projects/spock/build.gradle

```diff
@@ -1,14 +1,28 @@
 plugins {
-    id 'com.tngtech.jgiven.gradle-plugin'
+    id 'com.tngtech.jgiven.gradle-plugin' version "1.0.0-RC7"
+    id 'java-library'
 }
 
 apply plugin: 'groovy'
 
 description = "Module for writing JGiven tests with Spock"
 
+ext {
+    jGivenVersion = version
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
 dependencies {
-    implementation 'org.spockframework:spock-core:1.3-groovy-2.5'
-    testImplementation 'com.tngtech.jgiven:jgiven-spock:latest.integration'
+    implementation "org.spockframework:spock-core:2.0-M4-groovy-3.0"
+    implementation "org.spockframework:spock-junit4:2.0-M4-groovy-3.0"
+    testImplementation "com.tngtech.jgiven:jgiven-spock:${jGivenVersion}"
```

Comment:
how do the spock versions here relate to the one used by jgiven-spock?

Reply:
oh, those are super old. But, on second thought. Don't touch them. Renovatebot should ask us to update them soon enough

Reply:
I do not think that the current implementation of jgiven-spock is ready for spock2 that still not released

Reply:
@jsalinaspolo  
Oh my. That is a serious concern that you raise here. 
And indeed,  I hadn't noticed that the "M4" in the package name stands for milestone. 
Yet, I have already used a different PR to update the jgiven-spock package using the same `org.spockframework:spock-junit4:2.0-M4-groovy-3.0` glue module that is used here. I will however, go and double check that this change is not breaking (it obviously did not break the tests) 
I asked @fudler to have a look into splitting the jgiven-spock package into a jgiven-spock1 and jgiven-spock2 version, something that you seem to have started roughly a year ago. 
Maybe it is indeed more advisable to keep the current package at spock1 and use a spock2 package to gradually build up to spock2.